single_kernel_mongo/utils/helpers.py

```diff
@@ -73,3 +80,29 @@ def wrapper(self, *args, **kwargs):
 
 mongodb_only = partial(charm_kind_only, charm_kind=CharmKind.MONGOD)
 mongos_only = partial(charm_kind_only, charm_kind=CharmKind.MONGOS)
+
+
+def get_ldap_connection_status(state: LdapState) -> StatusBase:
```

Comment:
should this not be a part of the `get_status` function?

Reply:
This is called in the last case of the `get_status`, but exported in the helpers to reduce file size and function complexity, on top of providing an easily reusable function if needed.

Reply:
I think this is only used in the ldap module from within the `get_status` method, it would make sense to keep this method as part of the ldap manager instead of the helper